R/dash.R

```diff
@@ -573,10 +583,11 @@ Dash <- R6::R6Class(
           </body>
         </html>',
         private$name,
-        render_dependencies(depsCSS, local = private$serve_locally),
+        render_dependencies(depsCSS, local = private$serve_locally, prefix=self$config$routes_pathname_prefix),
```

Comment:
this should be `requests_pathname_prefix` - as the script tag is something that makes a request

Reply:
fixed in cc3ea048f7c2594e5efd137afcfb76324adba34d